rezerwacje-app/src/main/java/org/niewidoczniakademicy/rezerwacje/core/model/database/SystemUser.java

```diff
@@ -0,0 +1,34 @@
+package org.niewidoczniakademicy.rezerwacje.core.model.database;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.time.ZonedDateTime;
+
+@Data
+@Table
+@Entity
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class SystemUser {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NonNull
+    private String firstName;
+
+    @NonNull
+    private String lastName;
+
+    @NonNull
+    private String emailAddress;
+
+    @NonNull
+    private String phoneNumber;
+
+    @NonNull
+    private ZonedDateTime additionTime;
```

Comment:
Potrzebujemy tutaj `ZonedDateTime`? Może `LocalDateTime` by wystarczył, bo chyba obsługa stref czasowych nie będzie nam potrzebna

Reply:
A, jeszcze teraz tak się zastanawiam, bo skoro to jest klasa reprezentująca użytkownika, to nie powinna zawierać jakichś pól od danych logowania? Jakiś login i hasło?

Reply:
Done